samples/server/petstore/jaxrs-spec-interface-response/src/gen/java/org/openapitools/api/FakeApi.java

```diff
@@ -64,23 +73,23 @@
     @ApiOperation(value = "", notes = "For this test, the body for this request much reference a schema named `File`.", tags={ "fake",  })
     @ApiResponses(value = { 
         @ApiResponse(code = 200, message = "Success", response = Void.class) })
-    Response testBodyWithFileSchema(@Valid FileSchemaTestClass fileSchemaTestClass);
+    Response testBodyWithFileSchema(@Valid FileSchemaTestClass body);
 
     @PUT
     @Path("/body-with-query-params")
     @Consumes({ "application/json" })
     @ApiOperation(value = "", notes = "", tags={ "fake",  })
     @ApiResponses(value = { 
         @ApiResponse(code = 200, message = "Success", response = Void.class) })
-    Response testBodyWithQueryParams(@QueryParam("query") @NotNull    String query,@Valid User user);
+    Response testBodyWithQueryParams(@QueryParam("query") @NotNull    String query,@Valid User body);
```

Comment:
I noticed that the body parameter name changes from `user` to `body`. Likely you're not using the latest (or recent) master when creating this PR.

Please merge the latest master of the official repo into your branch and regenerate the sample again.

Reply:
I have used `4.3.x` as the origin of my branch, are you sure I should merge `master` in?

Reply:
OK. Please put it on hold. I'll take another look over the weekend or next week.